changelog.d/gh-6355.fixed

```diff
@@ -0,0 +1 @@
+Taint: Fixed a bug where taint labels were not being correctly applied to findings resulting from dot accesses. For instance, a taint pattern with a `requires` involving `x.y` might ignore the `requires` if the `y` produces a match.
```

Comment:
Even if just to make the review easier, we should hard wrap these files at 80 chars. It's painful having to scroll horizontally so much. Otherwise I need to open the file "raw" so that I can get soft wrapping by the browser.

Reply:
> For instance, a taint pattern with a `requires` involving `x.y` might ignore the `requires` if the `y` produces a match.
Maybe
> For instance, if a `pattern-source` that requires label `A` and adds label `B` matches a dot-access expression like `x.a`, the field `a` will get the label `B` even if it does not carry label `A` as required.
?